app/views/rubygems/show.html.erb

```diff
@@ -4,7 +4,11 @@
 <% content_for :head do %>
   <%= auto_discovery_link_tag(:atom, rubygem_versions_path(@rubygem, format: "atom"), {title: "#{@rubygem.name} Version Feed"}) %>
   <% if @rubygem.versions.any? && @latest_version.indexed %>
-    <link rel="canonical" href="<%= rubygem_version_url(@rubygem, @latest_version.slug) %>" />
+    <link rel="canonical" href="<%= rubygem_version_url(@rubygem, @latest_version.slug, params[:locale] ? { locale: } : {}) %>" />
```

Comment:
Why are we passing empty locale param to rubygem_version_url?

Reply:
@martinemde Thanks for your review. It should be `{ locale: params[:locale] } : {}` 👍

Reply:
fixed